lib/base-controller.js

```diff
@@ -4,8 +4,9 @@ var Controller = require('hmpo-form-wizard').Controller;
 var util = require('util');
 var _ = require('lodash');
 
-var BaseController = function BaseController() {
-  this.confirmStep = '/confirm';
+var BaseController = function BaseController(options) {
+  this.options = options || {};
```

Comment:
@JoeChapman: Edited to make you look silly.

PS: Yes, thanks for pointing that out!